### PR TITLE
Feature/css handles classes

### PR DESCRIPTION
**Pull request by gustavopvasconcellos**
#### What problem is this solving?

The lack of the classes on the name and image of product assemblies.

#### How to test it?

[Workspace](https://mcdon90--acctglobal.myvtex.com/mcproductrefid136/p)

#### Screenshots or example usage:

image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -116,6 +116,18 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+| `modalViewProductName` |
+| `modalViewProductInfos` |
+| `modalViewDoneButton` |
+| `productAssemblyOptionItemCustomize` |
+| `productAssemblyOptionItemCustomize__label` |
+| `productAssemblyOptionItemImage` |
+| `productAssemblyOptionItemName` |
+| `productAssemblyGroupName` |
+| `productAssemblyGroupNameRow` |
+| `productAssemblyGroupRequiredTag` |
```
Comment (guerreirobeatriz) [inline docs/README.md:130]: They should be added to the table in alphabetical order! Could you update this part, please? :) Thank you!

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (gustavopvasconcellos) [reply docs/README.md]: @guerreirobeatriz It's done! Could you continue the flow pls? 🙏

**Review by guerreirobeatriz — APPROVED** [review]

**Comment by kaisermann** [comment]
Hey @gustavopvasconcellos 👋 Thanks for your contribution! Can you resolve the file conflicts? Also, we've recently added the translation for the `Customize` string. Can you replace yours with it? https://github.com/vtex-apps/product-customizer/blob/master/messages/en.json#L29

**Comment by gustavopvasconcellos** [comment]
> Hey @gustavopvasconcellos 👋 Thanks for your contribution! Can you resolve the file conflicts? Also, we've recently added the translation for the `Customize` string. Can you replace yours with it? https://github.com/vtex-apps/product-customizer/blob/master/messages/en.json#L29

@kaisermann Its done! Thank you.

File: react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+
+const CSS_HANDLES = [
+  'modalViewProductContainer',
+  'modalViewProductImage',
+  'modalViewProductName',
+  'modalViewProductInfos',
+  'modalViewDoneButton',
+  'modalViewTitle',
+  'productAssemblyOptionItemCustomize',
+  'productAssemblyOptionItemCustomize__label',
```
Comment (kaisermann) [inline react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx:23]: Can you rename it to camelCase?

```suggestion
  'productAssemblyOptionItemCustomizeLabel',
```

**Review by kaisermann — CHANGES_REQUESTED** [review]
Can you also install your root dependencies and fix the lint errors on the files you've modified?

**Comment by gustavopvasconcellos** [comment]
> Can you also install your root dependencies and fix the lint errors on the files you've modified?

@kaisermann It's done! Tks

**Review by kaisermann — APPROVED** [review]

**Pull request merged**